src/styles/spacing/index.md.njk

```diff
@@ -106,20 +106,42 @@ The last part of the class represents the value you want to apply. For example,
 
 If you’re building your own components and want to reference the spacing scale directly in your SCSS file you can use the spacing scale through a mixin or a function.
 
+### Using the responsive spacing mixin
+
+To use the responsive spacing scale, include the [`govuk-responsive-margin`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-responsive-margin) or [`govuk-responsive-padding`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-responsive-padding) mixins.
+
+For example, to apply spacing unit 6 for 30px padding on large screens and 20px on small screens, use:
+
+```scss
+@include govuk-responsive-padding(6);
+```
+
+You can also add an argument to apply margin or padding in a single direction.
+
+For example, to apply spacing unit 6 for a 30px bottom margin on large screens and a 20px bottom margin on small screens, use:
+
+```scss
+@include govuk-responsive-margin(6, "bottom");
+```
+
 ### Using the static spacing function
 
-If you want to use static values that will not change based on breakpoints you can reference the static spacing scale function like this:
+If you want to apply 'large screen' spacing at all breakpoints, use the [`govuk-spacing` function](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-spacing) instead.
```

Comment:
I don't think the quote marks are needed here, given that the term large screen is defined earlier on in the page.

Reply:
Good catch, thanks @lfdebrux!

```suggestion
If you want to apply large screen spacing at all breakpoints, use the [`govuk-spacing` function](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-spacing) instead.
```